admin/src/assets/icons/ClipboardList.js

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+
+const ClipboardList = (props) => (
+  <svg width={20} height={20} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <path
+      d="M10 9.3a.7.7 0 0 0 0 1.4V9.3Zm2.5 1.4a.7.7 0 1 0 0-1.4v1.4ZM10 12.633a.7.7 0 0 0 0 1.4v-1.4Zm2.5 1.4a.7.7 0 1 0 0-1.4v1.4ZM7.5 9.3a.7.7 0 1 0 0 1.4V9.3Zm.009 1.4a.7.7 0 1 0 0-1.4v1.4ZM7.5 12.633a.7.7 0 1 0 0 1.4v-1.4Zm.009 1.4a.7.7 0 1 0 0-1.4v1.4Zm7.625-8.2v10h1.4v-10h-1.4ZM14.167 16.8H5.834v1.4h8.333v-1.4Zm-9.3-.967v-10h-1.4v10h1.4Zm.967-10.966H7.5v-1.4H5.834v1.4Zm6.666 0h1.667v-1.4H12.5v1.4ZM5.834 16.8a.967.967 0 0 1-.967-.967h-1.4A2.367 2.367 0 0 0 5.834 18.2v-1.4Zm9.3-.967a.967.967 0 0 1-.967.967v1.4a2.367 2.367 0 0 0 2.367-2.367h-1.4Zm1.4-10a2.367 2.367 0 0 0-2.367-2.366v1.4c.534 0 .967.432.967.966h1.4Zm-11.667 0c0-.534.433-.966.967-.966v-1.4a2.367 2.367 0 0 0-2.367 2.366h1.4ZM10 10.7h2.5V9.3H10v1.4Zm0 3.333h2.5v-1.4H10v1.4ZM9.167 3.2h1.667V1.8H9.167v1.4Zm1.667 1.933H9.167v1.4h1.667v-1.4Zm-1.667 0a.967.967 0 0 1-.967-.966H6.8a2.367 2.367 0 0 0 2.367 2.366v-1.4Zm2.633-.966a.967.967 0 0 1-.966.966v1.4A2.367 2.367 0 0 0 13.2 4.167h-1.4Zm-.966-.967c.534 0 .966.433.966.967h1.4A2.367 2.367 0 0 0 10.834 1.8v1.4ZM9.167 1.8A2.367 2.367 0 0 0 6.8 4.167h1.4c0-.534.433-.967.967-.967V1.8ZM7.5 10.7h.009V9.3H7.5v1.4Zm0 3.333h.009v-1.4H7.5v1.4Z"
+      fill="#9CA3AF"
```

Comment:
il faudrait rajouter les viewBox et les color
```suggestion
      fill="currentColor"
```

pour tous les svg qu'on importe, ils seront plus facilement réutilisables